test/utils_selection_tests.py

```diff
@@ -59,7 +59,7 @@ def make_last_write_date(server):
 def make_server_description(server, hosts):
     """Make a ServerDescription from server info in a JSON test."""
     server_type = server['type']
-    if server_type == "Unknown":
+    if server_type in ("Unknown", "PossiblePrimary"):
```

Comment:
Which spec change is this coming from?

Reply:
The `server_selection/ReplicaSetNoPrimary/read/PossiblePrimary.json` and `server_selection/ReplicaSetNoPrimary/read/PossiblePrimaryNearest.json` tests which have exists forever. Previously we just removed those test files manually after resyncing. Pymongo doesn't actually support a PossiblePrimary and doesn't need to support it. Now we run the tests because it's easy enough to convert PossiblePrimary to Unknown.

TLDR: this makes it easier to resync the server-selection spec tests.